code/site/components/com_pages/view/page/tmpl/default.html.php

```diff
@@ -9,8 +9,25 @@
 
 defined('KOOWA') or die; ?>
 
+<? // Before display - content plugin event ?>
+
+<? if($page->process->plugins) : ?>
```

Comment:
@johanjanssens,

When I was testing this, the event template helper requires `import_group` _(e.g. content or pages)_ in order to import the plugins that will be used. Since it's not declared here, no plugin will be triggered.

Reply:
Good catch, fixed.